apps/molgenis-components/package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@vue/compiler-sfc": "3.2.33",
     "axios": "0.27.2",
+    "lodash": "4.17.21",
```

Comment:
suggest to use js methods , if u really really really really need a method suggest to just import the single methode instead of the whole lib